src/test/fixtures/utils.js

```diff
@@ -293,8 +293,8 @@ export function fireFullClick(
   options?: FakeMouseEventInit
 ) {
   fireEvent(element, getMouseEvent('mousedown', options));
-  fireEvent(element, getMouseEvent('click', options));
   fireEvent(element, getMouseEvent('mouseup', options));
+  fireEvent(element, getMouseEvent('click', options));
```

Comment:
The order was wrong; As a result the change to the test wasn't working properly on the main branch. So I fixed that too.

Reply:
Thanks for the fix, not sure why I screwed it up.